documentation/operators/src/nodes/troubleshooting.md

```diff
@@ -308,10 +291,25 @@ Finally, restart your node. You should see if the mix node is using the port you
 
 ### What is `verloc` and do I have to configure my mix node to implement it?
 
-`verloc` is short for _verifiable location_. Mixnodes and gateways now measure speed-of-light distances to each other, in an attempt to verify how far apart they are. In later releases, this will allow us to algorithmically verify node locations in a non-fakeable and trustworthy manner.
+`verloc` is short for _verifiable location_. Mixnodes and gateways now measure speed-of-light distances to each other, in an attempt to verify how far apart they are. In later releases, this will allow us to algorithmically verify node locations in a non-fake-able and trustworthy manner.
 
 You don't have to do any additional configuration for your node to implement this, it is a passive process that runs in the background of the mixnet from version `0.10.1` onward.
 
+## Gateways & Network requesters
+
+### My gateway seems to be running but appears offline
+
+Check if your [firewall status](./maintenance.md#configure-your-firewall) is active and if the needed ports are allowed.
+
+### My gateway (with network requester) "is still not online..."
+
+Remember the epoch takes up to 1h. It's important to run your node right after bonding so it responds positively on our API ping tests. Otherwise it may be blacklisted.
+
+You may want to disconnect the network requester and let it run as a gatewy alone for some time to regain better routing score. 
+
+If it won't help, simply un-bond, delete the config and initialize a new one.
```

Comment:
Lets be careful about telling people to unbond in the future - as soon as we get delegated gateways then we want to make sure we remove this, as unbonding removes delegations

Reply:
deleted.

Correct we need to solve this on the dev side and not tell the solution is to erase the entire node and start a new one. Very good point. Brought it to the channel.